### ISSUE TITLE
Review Mindfulness Manifesto

### ISSUE THREAD

**Issue by stephen-rowan** [NFT-DAO/Governance-HOLON#28]
Review Mindfulness Manifesto contribution for incorporation into NFT-DAO Manifesto.

**Comment by stephen-rowan** [comment]
Hey Governance HOLON,

Dan has created a Pulll Request (a request to contribute changes) to **[Create "MindfulnessManifesto" #27](https://github.com/NFT-DAO/Governance-HOLON/pull/27)** 

It would be great if Governance HOLON can review and comment on Dan's request in this thread. I have cut and pasted a markdown version in this Comment to aid new users in reviewing Dan's text.

I have now created a Project for "NFT-DAO-Manifesto" here https://github.com/NFT-DAO/Governance-HOLON/projects/4.
The first issue (this thread) relates to Dan's Pull Request and I have assigned it to all members of Governance HOLON.

We also have the option of converting Dan's Pull Request into a GitHub Discussion if the HOLON decide against full merge of Dan's text (e.g. if we adopt only some of the text). 

All the best

Stephen

# Mindfulness Manifesto

We entertain the idea that this planet is a living organism, a holistic system – made of many decentralised parts - that strives for balance and sustainability from chaos, in order to create symbiotic, more efficient & less intrusive forms of life.

**We value life over profit**
In order to live in harmony with this planet, we will always favour:
* Complementing, over construing, the environment or individuals.
* Building on sovereign grade censorship resistant systems, offering data- & self-sovereignty for good actors. 
* Where possible create a harmonious habitat, blending in and supporting synergies with the environment.
* Preserving, over extracting, resources from the environment or individuals. Growing networks, over enterprises.
* Solutioning, over discussing problems.
* Cooperating, over competing.
* Open source ideas and build loyalty coalitions and teams (infosets/**Holons!**).
* Sustainability over quick profit
* Love, over fear of change.
* Embracing new technologies & fresh thought processes, over conserving faulty systems

**Comment by tttroyyy** [comment]
This is beautifully considered. Thank you Dan. 

Here, I would like to address the full Manifesto/Charter (doc. name pending) by also drawing on the Values/High-Level Concept section from the Business Plan, and Rich's Genesis document from the #Genesis-block-start-here Discord channel (attached). As a Charter is a document that defines our values, rights and privileges, it might pay to split the manifesto into the following sections: 

1) Introduction - Who We Are/Background (from the Genesis doc), 
2) Values: collated from the aforementioned docs and grouped under common themes, 
3) Community Guidelines: Code of Conduct/Rights/Priveleges/Expectations drawn from the Values section that members are expected to adhere to and afford others, 
4) Rules of Engagement: pertaining to how holons work and how members can expect to engage within them.

Sources

DAO Charter Template: https://medium.com/token-suisse/sample-dao-charter-c59f5396ceea
[GENESIS_OF_NFT_DAO.docx](https://github.com/NFT-DAO/Governance-HOLON/files/6261232/GENESIS_OF_NFT_DAO.docx)

**Comment by newmindflow** [comment]
Beautiful example Troy. I had prepared a comment but a careful reading of both the tone and content of the article you linked would render it unnecessary, as the example encapsulates my thoughts.

**Comment by stephen-rowan** [comment]
Hi Dan, I decided to merge your pull request instead of converting it to a Discussion.

I added a .md extension so it displays as markdown and moved it to /Governance-HOLON/05-Our-Values/Documents/Charter/ : https://github.com/NFT-DAO/Governance-HOLON/blob/main/05-Our-Values/Documents/Charter/MindfulnessManifesto.md). 

I will leave this Issue (https://github.com/NFT-DAO/Governance-HOLON/issues/28) Open as there is already a discussion thread active on it.

**Comment by DanM3rcurius** [comment]
Hi Stephen, 
thank you for adding the extension and teaching me something new! I was wondering why the formatting was off.
The idea to split the charter as mentioned by Troy sounds good.
How do you all feel about changing "Mindfulness Manifesto" to "Mindfulness Policy" ?

**Comment by tttroyyy** [comment]
Yes, Mindfulness Policy sounds good. A draft until we formalise some ratification process. It may be the inaugural first in our policies section. Another I'm thinking about is a process for triaging consensus levels (ie, ratification levels dependent upon holon-levels in the community). The exciting thing with some of these policies, is that we can consider how each one can evolve from social consensus to machine consensus (which will work for some, but not all policies - and which will be a policy in itself). My recurring question is: what parts of our processes can be automated going forward?

**Comment by DanM3rcurius** [comment]
"triaging consensus levels" what a brilliant term @tttroyyy ! 🧠 ⚡  
This gives me an idea for our "kudo" tokens: **How to build a verified domain expertise from within the community!** 
_And also start to have a piece of the "tool" to automate consensus / voting_

- Each Holon is represented by two sets of tokens, one internal and one external
- **Internal tokens** are eligible for voting on Holon decisions // _decisions encompassing all Holons are voted on by all Holon internal tokens_ 
- **external tokens:** Collectible by individuals outside Holon ; given by anyone from within the Holon for adding value in any form to the Holon 

**Requirements**

- The cost of minting all these tokens would be below 7 USD
- Daedalus wallet for every participant 
- a dev with a Cardano CLI setup to mint the tokens, or wait for me to have it figured out 😆 

@stephen-rowan Thanks for the hyphenheadsup 🙏

**Comment by stephen-rowan** [comment]
## Triage Idea

I have included Troy's **Triage** idea in [the Draft Charter](https://github.com/NFT-DAO/Governance-HOLON/blob/main/05-Our-Values/Documents/Charter/Draft-Charter.md) . Please post Comments on it under the Issue [Draft NFT-DAO Charter #29](https://github.com/NFT-DAO/Governance-HOLON/issues/29).

**Comment by DanM3rcurius** [comment]
here is a link to the tokenomics mindmap we're working on
[TOKENOMICS MINDMAP ON SAMEPAGE](https://samepage.io/app/#!/3b9bc08712435cebafcc8356a016413e2338ca7f/page-1020458760283237299
)

**Comment by jkochevar** [comment]
New to Github, so I'm sorry if this is in the wrong place, but I wanted to add a comment to the Mindfulness Manifesto. I see value in adding the idea: decentralization is localization.
